ports/stm32/ulpi.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2016 STMicroelectronics. All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ * USB3320 ULPI functions ported from stm32f7xx_lp_modes.c
+ *
+ * Source file: https://github.com/STMicroelectronics/STM32CubeF7/blob/v1.15.0/Projects/STM32746G-Discovery/Examples/PWR/PWR_CurrentConsumption/Src/stm32f7xx_lp_modes.c
+ */
```

Comment:
This file is very simple and does not need to carry the ST license/copyright.  No code in this file was taken from the ST example (the function names are different) so I think it's misleading to have the ST license/copyright.

Instead please use the standard MIT license, and put your copyright (or mine).